### PR TITLE
Working parameter tree

### PR DESCRIPTION
**Pull request by philipqueen**
The goal of this PR is a fully functioning parameter tree. The current main branch has a parameter tree widget, but the entered values are not passed to the processing functions they are meant for.

So far, the parameters appear to be working, but the "skip this section" options do not work. Implementing this second step will help fix [this issue](https://github.com/freemocap/freemocap/issues/358)

### CODE REVIEW

**Comment by philipqueen** [comment]
I've now verified the parameters ARE being passed to their respective pipelines, so the changes so far are functioning properly

**Comment by philipqueen** [comment]
This is working and has all discussed changes, ready for review whenever.

File: freemocap/core_processes/post_process_skeleton_data/gap_fill_filter_and_origin_align_skeleton_data.py [hunk 1]
```diff
@@ -1043,12 +1044,15 @@ def gap_fill_filter_origin_align_3d_data_and_then_calculate_center_of_mass(
...
     processed_skel3d_frame_marker_xyz = fill_gaps_in_freemocap_data(raw_skel3d_frame_marker_xyz, )
 
-    logger.info(
-        f"Filtering data at with a {order}th order, zero-lag, low-pass Butterworth filter with a cut-off frequency of {cut_off} Hz..."
-    )
-    processed_skel3d_frame_marker_xyz = butterworth_filter_skeleton(
-        processed_skel3d_frame_marker_xyz, cut_off, sampling_rate, order
-    )
+    if skip_butterworth_filter:
+        logger.info("Skipping butterworth filter...")
```
Comment (jonmatthis) [inline freemocap/core_processes/post_process_skeleton_data/gap_fill_filter_and_origin_align_skeleton_data.py:1048]: better to do 

```python
if skip_bool:
   return

[rest of code]
```

File: freemocap/core_processes/post_process_skeleton_data/gap_fill_filter_and_origin_align_skeleton_data.py [hunk 2]
```diff
@@ -1043,12 +1044,15 @@ def gap_fill_filter_origin_align_3d_data_and_then_calculate_center_of_mass(
     # Interpolate the data
     processed_skel3d_frame_marker_xyz = fill_gaps_in_freemocap_data(raw_skel3d_frame_marker_xyz, )
 
-    logger.info(
-        f"Filtering data at with a {order}th order, zero-lag, low-pass Butterworth filter with a cut-off frequency of {cut_off} Hz..."
-    )
-    processed_skel3d_frame_marker_xyz = butterworth_filter_skeleton(
-        processed_skel3d_frame_marker_xyz, cut_off, sampling_rate, order
-    )
+    if skip_butterworth_filter:
```
Comment (jonmatthis) [inline freemocap/core_processes/post_process_skeleton_data/gap_fill_filter_and_origin_align_skeleton_data.py:1047]: delete all this commented out stuff

**Review by jonmatthis — APPROVED** [review]
looking good!

**Pull request merged**